app/src/main/res/layout/fragment_item_detail.xml

```diff
@@ -13,14 +13,20 @@
         android:layout_height="match_parent"
         android:background="@color/backgroundGrey">
     <include layout="@layout/include_backable"/>
+    <include layout="@layout/fragment_warning"
+             android:layout_width="match_parent"
+             android:layout_height="wrap_content"
+             app:layout_constraintTop_toBottomOf="@id/toolbar"
+             app:layout_constraintBottom_toTopOf="@id/card_view"
+             tools:text="@string/no_internet_connection"/>
 
     <android.support.v7.widget.CardView
             xmlns:card_view="http://schemas.android.com/apk/res-auto"
             android:id="@+id/card_view"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:layout_gravity="center"
-            card_view:layout_constraintTop_toBottomOf="@id/toolbar">
+            card_view:layout_constraintTop_toBottomOf="@id/networkWarning">
```

Comment:
What happens when `networkWarning` disappears?

Reply:
It constrains to the "height=0" network warning